homeassistant/components/permobil/entity.py

```diff
@@ -0,0 +1,36 @@
+"""PermobilEntyty class."""
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import MyPermobilCoordinator
+from .sensor import PermobilSensorEntityDescription
+
+
+class PermobilEntity(CoordinatorEntity[MyPermobilCoordinator]):
+    """Representation of a permobil Entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: MyPermobilCoordinator,
+        description: PermobilSensorEntityDescription,
```

Comment:
Oh wait, you're using the SensorEntityDescription here, that's not corect. This shound be a generic one.

```suggestion
        description: EntityDescription,
```